components/cover/basic/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@schibstedspain/sui-cover-basic",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run babel && npm run sass",
+    "babel": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
```

Comment:
Ok. I was using 4.10 and now I have 4.11.

Note that `sui-studio link` is failing because it is trying to use `babel` instead of `build:js`